jenkins/deployment/community/community-build-cortx.groovy

```diff
@@ -19,9 +19,11 @@ pipeline {
         string(name: 'OS_VERSION', defaultValue: 'CentOS 7.9.2009 x86_64', description: 'Operating system version', trim: true)
         string(name: 'REGION', defaultValue: 'ap-south-1', description: 'AWS region', trim: true)
         string(name: 'KEY_NAME', defaultValue: 'automation-key', description: 'Key name', trim: true)
+        string(name: 'IMAGE_CLEANUP', defaultValue: 'no', description: 'Image Cleanup', trim: true)
```

Comment:
As discussed lets not add additional parameter

Reply:
We will add a parameter for now due to the known docker daemon issue. We will investigate this later.